habitat/tasks/rearrange/sub_tasks/articulated_object_sensors.py

```diff
@@ -50,7 +50,7 @@ def _get_observation_space(self, *args, **kwargs):
 
     def get_observation(self, observations, episode, *args, **kwargs):
         marker = self._task.get_use_marker()
-        ee_trans = self._sim.robot.ee_transform
+        ee_trans = self._sim.get_robot_data(self.robot_id).robot.ee_transform
```

Comment:
won't `self.robot_id` always be none and thus get_robot_data always return `robots_mgr[0]` ?

Reply:
Yes... this is true, but it is needed for future PR of multi-agent support. Robot manager was so entangled in all this code that it was hard for me to separate it out.